openapi/resource-server.yaml

```diff
@@ -1115,6 +1117,7 @@ components:
           description: The total amount successfully received (by all receivers) using the current outgoing payment grant.
         metadata:
           type: object
+          additionalProperties: true
```

Comment:
to allow generating 
```
{
            [key: string]: unknown;
        };
}
```
instead of `Record<string, never>`

Reply:
@mkurapov Missed one `"create-incoming-payment"` `requestBody`. 
<img width="762" alt="image" src="https://github.com/user-attachments/assets/dc77b27f-84d6-4797-af2c-ed3ad245fecb" />

Reply:
Thanks for the flag @sidvishnoi! added PR: #581